tests/cloner_test.go

```diff
@@ -98,14 +98,15 @@ var _ = Describe("all clone tests", func() {
 			fmt.Fprintf(GinkgoWriter, "INFO: %s\n", sourcePvcDiskGroup)
 			Expect(err).ToNot(HaveOccurred())
 
-			By("verifying pvc content")
-			sourceMD5, err := f.GetMD5(f.Namespace, pvc, diskImagePath, 0)
+			By("Calculating the md5sum of the source data volume")
+			sourceMD5, err := f.RunCommandAndCaptureOutput(utils.PersistentVolumeClaimFromDataVolume(dataVolume), "md5sum "+diskImagePath)
```

Comment:
I do not understand what is changed here. The sourcePvc returned by k8s was changed to sourcePvc specification create by helper function. Wich does not change much here. 

But I think the biggest change is the executor pod. Instead of using getMd5 which creates `verifierPod ` now it creates `execute-command` pod.
```
pvc, err := f.K8sClient.CoreV1().PersistentVolumeClaims(dataVolume.Namespace).Get(context.TODO(), dataVolume.Name, metav1.GetOptions{})
```
to this:
utils.PersistentVolumeClaimFromDataVolume(dataVolume)